src/components/TokenSelectors/AptosTokenPicker.tsx

```diff
@@ -16,12 +16,13 @@ type AptosTokenPickerProps = {
   tokenAccounts: DataWrapper<ParsedTokenAccount[]> | undefined;
   disabled: boolean;
   resetAccounts: (() => void) | undefined;
+  nft?: boolean;
 };
 
 const returnsFalse = () => false;
 
 export default function AptosTokenPicker(props: AptosTokenPickerProps) {
-  const { value, onChange, tokenAccounts, disabled } = props;
+  const { value, onChange, tokenAccounts, disabled, nft } = props;
```

Comment:
`nft` is optional, so we can do something like this to make `false` as default:
```suggestion
  const { value, onChange, tokenAccounts, disabled, nft = false } = props;
```